### DISCUSSION TITLE
initializer overrides prior initialization results in yellow squiggles

### DISCUSSION THREAD

**Discussion by hinxx** [microsoft/vscode-cpptools#8488] [Q&A]
In a C code I'm using temporary struct initializer for my function argument. In the editor I see a warning:

```
initializer overrides prior initialization of this subobject [-Winitializer-overrides]
```

The code compiles fine with GCC (no warnings or errors).

How do I make vscode not show the warning with yellow squiggles? I was hoping to find a place to supply `-Wno-initializer-overrides` but no idea where I should put it; I tried in several JSON files but nothing worked.

**Comment by sean-mcmanus** [comment]
How are you getting that warning to appear? Our extension normally shouldn't output that warning -- it appears to be coming from your compiler.

**Reply by hinxx** [reply]
Hmm, I'll need to doible check which extension would that be then. I might have some other extension that is c/c++ related. Thanks!
